run-your-own-gem-server.md

```diff
@@ -14,31 +14,36 @@ may have private code, internal to your organization, that you'd like to
 distribute and manage as gems without making the source publicly available.
 
 There are a few options to set up a server to host gems from within your
-organization. This guide covers the `gem server` command and the [Gem in a
-Box](https://github.com/geminabox/geminabox) project. It also discusses how to
-use these servers as gem sources during development.
+organization. This guide covers the [Gemstash](https://github.com/rubygems/gemstash)
+and [Gem in a Box](https://github.com/geminabox/geminabox) project. It also
+discusses how to use these servers as gem sources during development.
```

Comment:
The page also has Gemirro.

Reply:
Fixed by mentioning Gemirro in the intro paragraphs. :+1: